Makefile

```diff
@@ -116,8 +116,7 @@ goget-tools:
 
 .PHONY: goget-ginkgo
 goget-ginkgo:
-	@echo "This is no longer used."
-	@echo "Ginkgo can be executed directly from this repository using command '$(RUN_GINKGO)'"
+	 go install github.com/onsi/ginkgo/v2/ginkgo@latest
```

Comment:
Just curious, using the vendored version no longer works with this upgrade?
While I understand the previous behavior of running Ginkgo from the vendor folder (making sure the right version of Ginkgo is running), I think installing the `latest` version might be dangerous.

Reply:
When we are trying to use it the old way using vendor, it was not recognizing the new flags, and was not honoring the old flags.
I think we can use vendor after we merge [PR](https://github.com/redhat-developer/odo/pull/5833) in.

Reply:
I will update that with a specific version, thanks.

Reply:
I think it will make this PR huge. I can create a new PR replacing your #5833. We can merge this PR after the cleanup PR if merged?
WDYT @feloy